nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -334,6 +335,26 @@ public static AccessPath getAccessPathForNode(
       return fromFieldAccess((FieldAccessNode) node, apContext);
     } else if (node instanceof MethodInvocationNode) {
       return fromMethodCall((MethodInvocationNode) node, state, apContext);
+    } else if (node instanceof ArrayAccessNode) {
+      return fromArrayAccess((ArrayAccessNode) node, apContext);
+    } else {
+      return null;
+    }
+  }
+
+  @Nullable
+  private static AccessPath fromArrayAccess(ArrayAccessNode node, AccessPathContext apContext) {
+    return fromNodeAndContext(node, apContext);
+  }
+
+  @Nullable
+  private static Element getElementFromArrayNode(Node arrayNode) {
+    if (arrayNode instanceof LocalVariableNode) {
+      return ((LocalVariableNode) arrayNode).getElement();
+    } else if (arrayNode instanceof FieldAccessNode) {
+      return ((FieldAccessNode) arrayNode).getElement();
+    } else if (arrayNode instanceof MethodInvocationNode) {
+      return ASTHelpers.getSymbol(((MethodInvocationNode) arrayNode).getTree());
```

Comment:
Allowing the array itself to be the result of a method invocation is a bit more risky.  Let's remove this case, and handle it later if we decide we need it